src/app/app.component.ts

```diff
@@ -8,8 +8,14 @@ import { NgxPermissionsService } from 'ngx-permissions';
 })
 export class AppComponent {
   title = 'deploji';
+  theme = localStorage.getItem('theme') ? localStorage.getItem('theme') : 'light-theme';
 
   constructor(private authService: AuthService, private permissionService: NgxPermissionsService) {
     this.permissionService.loadPermissions(this.authService.permissions);
   }
+
+  setTheme(theme) {
+    this.theme = theme;
```

Comment:
Moim zdaniem lepiej to przenieść do backendu. Jak zaloguje się na innym kompie to będę miał wtedy tak ustawione